src/Storages/Distributed/DirectoryMonitor.cpp

```diff
@@ -332,7 +332,7 @@ namespace
 
     uint64_t doubleToUInt64(double d)
     {
-        if (d >= std::numeric_limits<uint64_t>::max())
+        if (d >= double(std::numeric_limits<uint64_t>::max()))
```

Comment:
What motivated to make this change?
If we can enable some new warning or clang-tidy check, let's do that.

Reply:
The clang-tidy rule is `cppcoreguidelines-narrowing-conversions`. It might be too strict for us now.